tools/twix/src/repository_parameters.rs

```diff
@@ -80,7 +95,7 @@ fn last_octet_from_ip_address(ip_address: Ipv4Addr) -> u8 {
 }
 
 // Create tree structure from path and value points to the last key i.e. a.b.c -> { a: { b: { c: value } } }
-fn nest_value_at_path(path: &str, value: &Value) -> Value {
+fn make_sparse_value_tree_from_path(path: &str, value: &Value) -> Value {
```

Comment:
What was the reasoning to change the name?

Reply:
With the merge and prune operations done, it felt clearer to refer this as a sparse tree, but in retrospection I think this might not convey what this parameter does better than the old one.
What's your opinion? I can revert if you think the same.

Reply:
Interestingly, during the refactorings back then to your that I did, I once renamed it from `make_sparse_value_tree_from_path()` to `nest_value_at_path()`. You are now changing it back. :joy:

What I find suboptimal with the *make*-function is that it does not describe how the value tree is created. I thought that the *nest*-function better describes that you're just putting this value at a specific depth and the tree nodes are named after path segments. :thinking: